City-of-Helsinki/hel-fi-drupal-grants
# feat: AU-2389: E2E, Changed the edit-acting-year field in every test to always be dynamic

# [AU-2384](https://helsinkisolutionoffice.atlassian.net/browse/AU-2384)

## What was done

This pull request changes the `edit-acting-year` field in every application so that the first available year in the fields drop-down is always selected instead of a hard-coded one. This change was needed since our tests would sometimes fail if the hard-coded year in the test data was not available on the actual application we were testing.

## How to install

Make sure your instance is up and running on correct branch.
* `feature/AU-2389-e2e-edit-acting-year-fix`
* `make fresh`
* `make drush-cr`

## How to test

- Start by setting `ENABLED_FORM_VARIANTS="draft"` in your `.env` file.

- Run a few different test to make sure the changed to `edit-acting-year` field work: `make test-pw-p PROJECT=forms-48-registered forms-51-registered forms-60-registered`

[AU-2384]: https://helsinkisolutionoffice.atlassian.net/browse/AU-2384?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
